src/utils/page.js

```diff
@@ -78,8 +78,13 @@ async function takeNewScreenshotsOfPreviews (page, previewMap, { dir, progress,
   }
 }
 
-async function takeNewScreenshotOfPreview (page, preview, index, actionState, { dir }) {
+async function takeNewScreenshotOfPreview (page, preview, index, actionState, { dir, delay: wait }) {
```

Comment:
Why `delay: wait`? Shouldn't it just be `wait`?

Reply:
Sure, sorry for that. I named option "delay" at first, but then renamed it to "wait" to make it consistent with the wait option in custom actions.

Reply:
Just pushed a fix